fabtests/benchmarks/benchmark_shared.c

```diff
@@ -311,11 +325,11 @@ int bandwidth(void)
 						opts.transfer_size, NO_CQ_DATA,
 						tx_ctx_arr[j].buf, tx_seq);
 			} else {
-				ret = ft_post_tx_buf(ep, remote_fi_addr,
```

Comment:
I feel like we should keep the fi more test only an opt-in feature, that we should still keep the non-fi-more test using ft_post_tx_buf